src/common/components/DialogDescription/DialogDescription.theme.ts

```diff
@@ -0,0 +1,20 @@
+import { type VariantProps, tv } from "tailwind-variants";
+
+export type DialogDescriptionVariants = VariantProps<
+  typeof dialogDescriptionTheme
+>;
+
+export const dialogDescriptionTheme = tv(
+  {
+    slots: {
+      dialogDescription: ["text-muted-foreground", "text-sm"],
+    },

```

Comment:
Should we just directly use the base key?

```suggestion
    base: ["text-muted-foreground", "text-sm"],
```

Reply:
Same goes for all the Dialog composition components. Wdyt?

Reply:
sure! will add them in